src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -70,9 +80,11 @@
 (handlers/register-handler-fx
   :update-balance-fail
   (fn [_ [_ err]]
-    (.log js/console "Unable to get balance: " err)))
+    (.log js/console "Unable to get balance: " err)
+    (dispatch [:wallet/error-message :error])))
```

Comment:
This is the very thing which in the long runs leads to problems with long even chains. There is no reason why updating balance should trigger chain of 2 events, when only 1 external input is involved. 
Please pull up functionality in the event `:wallet/error-message` into plain function and use it directly as function call in `:update-balance-fail` so we don't unnecessary dispatch 2 events when 1 will do totally fine.

Reply:
@janherich Good point; should've spotted that. Can't see a reason the `db assoc` couldn't happen in `update-balance-fail`.